internal/pkg/manifest/backend_svc.go

```diff
@@ -62,6 +63,17 @@ func NewBackendService(props BackendServiceProps) *BackendService {
 		svc.BackendServiceConfig.TaskConfig.Memory = aws.Int(MinWindowsTaskMemory)
 	}
 	svc.parser = template.New()
+	for _, envName := range props.PrivateOnlyEnvironments {
```

Comment:
Can we make it a function so that we don't repeat ourselves? or like i mentioned earlier we can make `PrivateOnlyEnvironments` a field not dedicated to private subnets setting only and then we can have a function to apply the env override setting to the manifest struct.